packages/shared/lib/participation/stores.ts

```diff
@@ -207,6 +207,24 @@ export const shimmerStakingRewards: Readable<number> = derived(participationOver
     sumStakingRewards(StakingAirdrop.Shimmer, overview)
 )
 
+/**
+ * The total accumulated rewards for selected account
+ */
+export const selectedAccountStakingRewards: Readable<{ assembly: number; shimmer: number }> = derived(
+    participationOverview,
+    (overview) => {
+        const selectedAccountOverview = overview.find(
+            (accountOverview) => accountOverview.accountIndex === get(selectedAccountId)
```

Comment:
Will this ever find? accountIndex is an index number, and selectedAccountId is the string ID of the account, so I dont think this can work

Reply:
imo this store should be
```ts
export const selectedAccountStakingRewards: Readable<{ assembly: number; shimmer: number }> = derived(
    [participationOverview, selectedAccountId],
    ([$participationOverview, $selectedAccountId]) => {
        const selectedAccountOverview = $participationOverview.find(
            (accountOverview) => accountOverview.accountIndex === get(selectedAccount)?.index
        )
        if (selectedAccountOverview) {
            return {
                assembly: selectedAccountOverview.assemblyRewards + selectedAccountOverview.assemblyRewardsBelowMinimum,
                shimmer: selectedAccountOverview.shimmerRewards + selectedAccountOverview.shimmerRewardsBelowMinimum,
            }
        }
    }
)
```
but it shouts a circular dependency, so we need to move this around 🥲

Reply:
Im not making it derived depending on `selectedAccountId` so it only triggers when the ID changes, and not the account itself which can change a lot while selected, but we can also make this derived store dependent of `selectedAccount`